agents/atd-data03-docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:22.04
+ENV TZ="America/Chicago"
+RUN apt-get update
+RUN apt-get update && \
+    apt-get install -yq tzdata && \
+    ln -fs /usr/share/zoneinfo/America/Chicago /etc/localtime && \
+    dpkg-reconfigure -f noninteractive tzdata
+RUN apt-get upgrade -y
+RUN apt-get install -y vim magic-wormhole aptitude python3-pip docker.io black
+COPY ./requirements.txt /root/requirements.txt
```

Comment:
Thanks for making this Frank! 
Curious, I see we're our own `requirements.txt` file for Prefect, are we using that over the [base prefect images](https://hub.docker.com/r/prefecthq/prefect) because we're worried that a prefect update could cause things to break?

Reply:
This is a good question - this requirements.txt actually is used to put python libraries (such as `prefect`) on the host that runs the agent, which is going to be a docker container. That agent itself will fire up the docker image you create and push to the registry during flow registration. It's two layers deep of docker containers logically, but in practice, they all are running as siblings on the host container thanks to the passing down of the docker daemon control socket.